### ISSUE TITLE
[Checkbox] Require to Hit VO Keys Twice to Move to Next Item when Using Checkbox in Table

### ISSUE THREAD

**Issue by Lijing29** [OfficeDev/office-ui-fabric-react#11400]
### Environment Information

- **Package version(s)**: version 7
- **Browser and OS versions**: VoiceOver + Safari

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/lijinglin29/pen/QWwwQJJ

#### Actual behavior:

When using VO + Right Arrow to navigate items in the table, it requires to hit VO + Right Arrow key twice to move from a item containing to next item.
When navigate to a checkbox item, it would announce "Row 2 of 3 header 1, a1, unchecked checkbox, Column 1 of 2". 
![Screen Shot 2019-12-08 at 6 30 29 PM](https://user-images.githubusercontent.com/50400285/70402616-fc4f4680-1a6e-11ea-87d3-d03c5847459b.png)
Hit VO + Right Arrow, it would announce "a1, unchecked, checkbox".
![Screen Shot 2019-12-08 at 6 31 07 PM](https://user-images.githubusercontent.com/50400285/70402622-02ddbe00-1a6f-11ea-97b7-6687b0e355db.png)

#### Expected behavior:

When using VO + Right Arrow to navigate an item containing checkbox in a table, hit VO + Right Arrow key once should move to the next item.

**Comment by micahgodbolt** [comment]
Does this issue repro in Narrator? Also, does this behavior happen with a normal input? Or only with our checkbox.

**Comment by joschect** [comment]
@Lijing29 I am not able to reproduce this on a mac with voice over using the codepen provided. It moves selection immediately on VO + Right arrow. Could you confirm that this is still happening? If it is happening, then it's likely caused by a difference in os/voice over version and is not a fabric bug. To confirm that, could you test using this basic version [codepen](https://codepen.io/slimemold/pen/gObPemW)?

**Comment by Lijing29** [comment]
@joschect The issue would not happen in the [codepen](https://codepen.io/slimemold/pen/gObPemW) you provided, as in your codepen, the checkbox is implemented based on input tag with type='checkbox' or checkbox tag. However, the issue I mentioned it would only happen in the checkbox based on checkbox component provided by Fabric UI.

**Comment by Lijing29** [comment]
@micahgodbolt This issue could not be reproed in Narrator. And this behavior also would not happen with a normal input.
In other words, this issue only happen in the checkbox provided by Fabric UI with VoiceOver + Safari when the checkbox is in a table.

**Comment by joschect** [comment]
@Lijing29 To clarify, I am not able to reproduce the problem in either codepen, the one you provided or the one that I provided. To help us narrow things down, what version of macos are you on?

**Comment by Lijing29** [comment]
@joschect The version of macOS is **macOS mojave 10.14.6**, and the version of Safari is **13.0.3**.

**Comment by joschect** [comment]
@Lijing29 I do seem to be behind in both OS and safari version. I wonder if mac broke something. I'll have to update and try again. Given that it works in one version, and is broken in another, I'm guessing that this is a bug in the Mac narrator/safari especially because the actual checkbox that's receiving focus in the fabric checkbox is a native input with role="checkbox"

**Comment by Lijing29** [comment]
@joschect Based on your investigation, the bug appears in some version of Mac. Would we like to ask Mac for help? I would like to know what I can do here. Thanks.

**Comment by joschect** [comment]
@Lijing29 You can see if you can file a bug against Mac VO. As far as I can tell our checkbox should be WCG 1.1 accessible.  @jamesedjac or @betrue-final-final might be able to confirm that our checkbox is correctly labeled and interactable.

**Comment by khmakoto** [comment]
Gentle ping for @jamesedjac and @betrue-final-final.

**Comment by Lijing29** [comment]
If we add **aria-hidden='true'** in **label** tag (in green rect), the issue would be solved and it would not affect Narrator on Edge/IE and VoiceOver on Chrome. 
![image](https://user-images.githubusercontent.com/50400285/71861032-e006fe00-312f-11ea-8034-ecbec6be6397.png)
@joschect @jamesedjac @betrue-final-final Is it possible for you to fix this bug using this method?

**Comment by Lijing29** [comment]
@joschect @jamesedjac @betrue-final-final Could you please help to check whether the fix method I have proposed before works or not? Thanks.

**Comment by joschect** [comment]
@lidermanrony I'm a bit confused, that label is expected to get read out, I think. Aria-hidden is for something that shouldn't be used to provide screenreader information, something like a decorative image which isn't related to the text.

**Comment by xugao** [comment]
this seems to be a bug with Apple VO.
